cmd/drafter-peer/main.go

```diff
@@ -46,6 +46,7 @@ func main() {
 	gid := flag.Int("gid", 0, "Group ID for the Firecracker process")
 	enableOutput := flag.Bool("enable-output", true, "Whether to enable VM stdout and stderr")
 	enableInput := flag.Bool("enable-input", false, "Whether to enable VM stdin")
+	enableInputKeepalive := flag.Bool("enable-input-keepalive", true, "Whether to continuously write backspace characters to the VM stdin to force the VM stdout to flush")
```

Comment:
Do you have a sense of any other potential side effects of enabling this? For example, sending characters in a tight loop could cause many interruptions for the process to handle, potentially affecting performance during a migration or other CPU-intensive operations. Maybe it would be better as opt-in instead of opt-out?